modules/exploits/unix/http/zivif_ipcheck_exec.rb

```diff
@@ -0,0 +1,88 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'             => "Zivif Camera Web Interface Blind Remote Command Execution ",
+      'Description'      => %q{
+          This module exploits a remote command execution vulnerability in Zivif 
+       webcams.  This is known to impact versions prior to and including v2.3.4.2103.
+       Exploit was reported in CVE-2017-17106.
+      },
+      'License'          => MSF_LICENSE,
+      'Author'           => [ 'Silas Cutler (p1nk)' ],
+      'References'       =>
+        [
+          [ 'URL', 'https://nvd.nist.gov/vuln/detail/CVE-2017-17106' ],
```

Comment:
Redundant due to `CVE` reference.

```suggestion
```

Perhaps:

```suggestion
          [ 'URL', 'https://seclists.org/fulldisclosure/2017/Dec/42' ],
```